### PR TITLE
The fucntionality of Add and Remove is Added

### PR DESCRIPTION
**Pull request by Nizamuddin4493**
- Removed all hardcoded items from the tasks array.
- Created a new JavaScript file for the new functionality.
- Implemented a function for adding a new task (add a new element to the array).
- Implement a function for deleting a task (remove an element from the array).
- Implement a function for editing task descriptions.

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -10,20 +10,33 @@
...
+        </form>
       </div>
       <hr>
         <div class="parent-node">
           <!-- To Do List items are display here. -->
+          <!-- <div class="show-list">
+            <input type="checkbox" class="check-box">
+            <p class="tast">asdfsadfsadfs</p>
+            <img src="images/drag-drop.png" alt="" class="drag-drop">
+          </div> -->
```
Comment (Cathella) [inline src/index.html]: - [ ] Please remove all commented blocks of code because they usually make code unreadable

File: src/activeList.js [hunk 2]
```diff
@@ -0,0 +1,151 @@
+/* eslint-disable no-unused-vars */
+/* eslint-disable import/no-mutable-exports */
+/* eslint-disable no-use-before-define */
+/* eslint-disable no-restricted-globals */
```
Comment (Cathella) [inline src/activeList.js]: You should not disable the linters, they provide a standard for team/community code here at Microverse and in your future workplaces. 

- [ ] Please kindly fix all the linter errors instead.

**Review by Cathella — CHANGES_REQUESTED** [review]
## 🟡 Status: Changes Required 🛑

### Hi @Nizamuddin4493 👋🏻, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>

## To Highlight 👏🏻

✔️ No linter errors.
✔️ You used correct Github Flow.
✔️ Professional README.
✔️ Used webpack to bundle JavaScript.
✔️ Implemented adding a new task
✔️ Implemented editing task descriptions
✔️ Implemented deleting a task
✔️ Removed hard-coded items from the tasks array
✔️ Used local storage to store To-Do List changes

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Nizamuddin4493** [comment]
## Hi Dear Reviewer.
#### With most respect, I would like to acknowledge that I have solved some of eslint errors but some of them are out of my knowledge range that is because I have bundled the js-files to one js-file from which the dist's file is updated. but to bundles of js-files to one js-file, It just generates an eslint error of unused variable, no-restricted-globals etc. that's why I disable the eslint. I searched a lot but failed to find any solution for this.

**Review by ZeenatLawal — APPROVED** [review]
Hi @Nizamuddin4493, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**